ios/RCTTWVideoModule.m

```diff
@@ -273,11 +281,18 @@ - (void)startCameraCapture:(NSString *)cameraType {
 }
 
 RCT_EXPORT_METHOD(stopLocalVideo:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    if (! self.isCameraCaptureActive) {
```

Comment:
This is the key part of the fix: prevents the call to stopCaptureWithCompletion which would hang if a capture is not currently active.  That caused the flipCamera() sequence to get stuck at that stage.